src/runtime/particle-spec.ts

```diff
@@ -77,7 +77,7 @@ export class ConnectionSpec {
   }
 }
 
-type SerializedSlotSpec = {
```

Comment:
Should probably open up a bug to add docs for newly exported classes..

Reply:
I removed the extra exports - they predate this PR and are irrelevant (only added them to get rid of annoying ide warnings, but there is a todo in this file to resolve them already)